claim/README.md

```diff
@@ -83,21 +97,38 @@ can trust the endpoint if necessary.
 Claiming can be performed through a SOCKS5 proxy. To do this when calling the script directly supply the proxy
 endpoint as:
 
-```
-netdata-claim.sh -token=MYTOKEN1234567 -rooms=room1,room2 -proxy=socks5h://127.0.0.1:11081
+```bash
+netdata-claim.sh -proxy=socks5h://127.0.0.1:11081 -token=MYTOKEN1234567 -rooms=room1,room2
 ```
 
-When claiming via the `netdata` binary set the following options in the config:
-```
+When claiming via the `netdata` binary, set the following options in your `netdata.conf` file:
```

Comment:
`When claiming via the `netdata` binary, set the following options in your `netdata.conf` file`
add
`in case you need to use proxy to connect to internet:` ?